geopandas/io/file.py

```diff
@@ -236,14 +236,20 @@ def _read_file(filename, bbox=None, mask=None, rows=None, **kwargs):
                 f_filt = features
             # get list of columns
             columns = list(features.schema["properties"])
+            datetime_fields = [
+                k for (k, v) in features.schema["properties"].items() if "datetime" in v
```

Comment:
Regarding the `"datetime" in v`, can this be `v == "datetime"`? Or are there multiple types of "datetime"?

Reply:
> I think that Fiona should also support reading timezones. Did you test that?
No - I noted I probably should in my original comment when I made the PR. I guess testing that would make `pytz` a test only dependency? At a quick glance I guess that's kind of similar to the use of `fsspec`, and I would just need to add pytz to the `ci/envs/*.yaml`s

>Or are there multiple types of "datetime"?
No, I think what I have is just imprecise, will switch that too.

Reply:
> I guess testing that would make pytz a test only dependency?

Pytz is a required dependency of pandas, so it should always be available